packages/core/core/src/resolveOptions.js

```diff
@@ -69,7 +69,10 @@ export default async function resolveOptions(
       ? path.resolve(outputCwd, initialOptions.cacheDir)
       : path.resolve(projectRoot, DEFAULT_CACHE_DIRNAME);
 
-  let cache = new Cache(outputFS, cacheDir);
+  let cache =
+    outputFS instanceof NodeFS
+      ? new LMDBCache(cacheDir)
+      : new FSCache(outputFS, cacheDir);
```

Comment:
Could this be an option from the CLI? `initialOptions.cache ?? ...`

Reply:
This is pretty similar to what @padmaia and I did with the remote cache experiment 😄 

https://github.com/parcel-bundler/parcel/commit/24211528a2d0e943e3b388bbd549df28ae646a80#diff-e813cdfb4bf5843d09ba22be43afb34fec2547aa41a896f743180f46833efcffR63